lib/tasks/create_test_efile_submissions.rake

```diff
@@ -10,7 +10,7 @@ namespace :efile do
 
     queued_group = preparing_group.take(75)
     queued_group.each do |submission|
-      submission.transition_to!(:queued, seeding: true)
+      submission.transition_to!(:queued)
```

Comment:
Does this mean that these rake-baed submissions will eventually try to get e-filed on my development workstation, if I have the credentials set up properly? If so, should we make a way to skip them, or should we just have fun sending them to the IRS?

Reply:
they fail because the test data isn't set up correctly because its a bit outdated, which im currently treating as a feature so that it prevents them from building and submitting properly 😬